app/views/entities/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<header id="header-main">
```

Comment:
- [ ] Kindly make a "Back" button (<) that navigates to the transactions page for that category as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=When%20the%20user%20clicks%20on%20the%20%22Back%22%20button%20(%3C)%2C%20the%20user%20navigates%20to%20the%20transactions%20page%20for%20that%20category.) instead of the home page.